crates/sui-cost-tables/src/tables.rs

```diff
@@ -0,0 +1,1120 @@
+            true,
+        ),
+        //
+        // Ref
+        //
+        (WriteRef, GasCost::new(2, 0), true),
+        // Performs c aopy
+        (ReadRef, GasCost::new(3, 1), true),
+        // Same cost as NOP. No work is done
+        (FreezeRef, GasCost::new(1, 1), false),
+        //
+        // Local access
+        //
+
+        // This performs a copy hence scales with size
+        (CopyLoc(0), GasCost::new(3, 1), true),
+        //
+        // TODO
+        // Values are approx
+        (MoveLoc(0), GasCost::new(1, 1), false),
+        (StLoc(0), GasCost::new(1, 1), false),
+        (CastU8, GasCost::new(2, 1), true),
+        (CastU64, GasCost::new(1, 1), true),
+        (CastU128, GasCost::new(1, 1), true),
+        (Branch(0), GasCost::new(1, 0), false),
+        (BrFalse(0), GasCost::new(1, 0), false),
+        (BrTrue(0), GasCost::new(1, 0), false),
+        // We have to modify the Move code to truly calculate this
+        // For now scale by size
+        (
+            Call(FunctionHandleIndex::new(0)),
+            GasCost::new(100, 1),
+            true,
+        ),
+        (
+            CallGeneric(FunctionInstantiationIndex::new(0)),
+            GasCost::new(582, 1),
+            false,
+        ),
+        (Ret, GasCost::new(638, 1), false),
+        (Nop, GasCost::new(1, 1), false),
+        (Abort, GasCost::new(1, 1), false),
+        //
+        // Not supported section
+        // These will not be implemented as they should never be triggered
+        //
+
+        // Not supported yet in Sui Move
+        (
+            Exists(StructDefinitionIndex::new(0)),
+            GasCost::new(41, 1),
+            false,
+        ),
+        (
+            ExistsGeneric(StructDefInstantiationIndex::new(0)),
+            GasCost::new(34, 1),
+            false,
+        ),
+        (
+            MoveTo(StructDefinitionIndex::new(0)),
+            GasCost::new(13, 1),
+            false,
+        ),
+        (
+            MoveToGeneric(StructDefInstantiationIndex::new(0)),
+            GasCost::new(27, 1),
+            false,
+        ),
+        (
+            MoveFrom(StructDefinitionIndex::new(0)),
+            GasCost::new(459, 1),
+            false,
+        ),
+        (
+            MoveFromGeneric(StructDefInstantiationIndex::new(0)),
+            GasCost::new(13, 1),
+            false,
+        ),
+        (
+            MutBorrowGlobal(StructDefinitionIndex::new(0)),
+            GasCost::new(21, 1),
+            false,
+        ),
+        (
+            MutBorrowGlobalGeneric(StructDefInstantiationIndex::new(0)),
+            GasCost::new(15, 1),
+            false,
+        ),
+        (
+            ImmBorrowGlobal(StructDefinitionIndex::new(0)),
+            GasCost::new(23, 1),
+            false,
+        ),
+        (
+            ImmBorrowGlobalGeneric(StructDefInstantiationIndex::new(0)),
+            GasCost::new(14, 1),
+            false,
+        ),
+        // Not supported yet in Move
```

Comment:
There is no syntax to call it with a non-zero size, but the bytecode instruction is still used

Reply:
How do I trigger it so I can calibrate it?

Reply:
Okay I will add it to the calibration list.